### PR TITLE
did homework with a description of the worker

### PR DESCRIPTION
**Pull request by senya2006**

### CODE REVIEW

File: Homework_13/Task_1.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    def name(self):
+        return self._name
+
+    @name.setter
+    def name(self, value):
+        if not isinstance(value, str):
+            raise ValueError("Name must be a string")
+        self._name = value
+
+    def __str__(self):
```
Comment (MarinaZelenska) [inline Homework_13/Task_1.py]: this method better situated after __init__

File: Homework_13/Task_1.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        return f'Worker: {self.name}, Position: {self.position}, Salary: ${self.salary}'
+
+# Example usage:
+worker1 = Worker("Vasya", "Cleaner", 300)
```
Comment (MarinaZelenska) [inline Homework_13/Task_1.py]: if __name__ == '__main__':

File: Homework_13/Task_1.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
+class Worker:
+    def __init__(self, name, position, salary):
+        self._name = name
+        self.position = position
+        self.salary = salary
```
Comment (MarinaZelenska) [inline Homework_13/Task_1.py]: In my view salary better change to __salary

**Review by MarinaZelenska — APPROVED** [review]

**Comment by senya2006** [comment]
fixed

**Pull request merged**